src/NuGet.Core/NuGet.Protocol/Repository.cs

```diff
@@ -46,7 +46,9 @@ public virtual IEnumerable<Lazy<INuGetResourceProvider>> GetCoreV3()
                 yield return new Lazy<INuGetResourceProvider>(() => new DownloadResourcePluginProvider());
                 yield return new Lazy<INuGetResourceProvider>(() => new DownloadResourceV3Provider());
                 yield return new Lazy<INuGetResourceProvider>(() => new MetadataResourceV3Provider());
+#pragma warning disable 612
```

Comment:
Use GlobalSupressions instead.

Reply:
Problem was VS was not offering any option to suppress this into GlobalSuppressions file. It seems other soIution is have to suppress for whole GetCoreV3() method or this file.